code/src/java/pcgen/output/actor/FactKeyActor.java

```diff
@@ -40,6 +42,9 @@
  */
 public class FactKeyActor<T> implements OutputActor<CDOMObject>
 {
+	private static final ObjectWrapperFacet WRAPPER_FACET = FacetLibrary
```

Comment:
Spring still has to instantiate the classes it manages. So if class A is managed by spring and A refers to FactKeyActor, then the startup will fail. Even if it works today, it still creates a startup dependency order which will be lying in wait for us to trip over in the future.